extensions/amp-consent/0.1/amp-consent.js

```diff
@@ -1035,7 +1035,10 @@ export class AmpConsent extends AMP.BaseElement {
 
     // Set up the __tcfApiLocator window to signal postMessage support
     const iframe = this.element.ownerDocument.createElement('iframe');
-    iframe.setAttribute('title', this.element.title || 'Consent Banner');
+    iframe.setAttribute(
```

Comment:
Should this be removed, if this iframe is not useful to the user?

Reply:
I'm not sure the iframe is not useful for the user. The iframe is used for setting up the __tcfApiLocator so this change is only trying to reflect that correctly in the iframe title, as the Consent Banner iframe is really the one setup in the consent-ui file.

Reply:
Sorry. I meant that this iframe is not user-accessible, and therefore it serves no purposes to set its title attribute. We should simply revert https://github.com/ampproject/amphtml/commit/5e5106bdcf2db0656b86539252f6d4f5ffc533be by removing this row altogether.

Reply:
Sorry, I miss interpreted your comment. Yes, that makes sense, great catch. I've removed it with 53fa8ba5dfd4acf0aee3be2694edf974334dfd4e. Thanks!